edkrepo/commands/list_repos_command.py

```diff
@@ -45,92 +47,117 @@ def get_metadata(self):
                      'positional': False,
                      'required': False,
                      'help-text': arguments.ARCHIVED_HELP})
+        args.append({'name': 'format',
+                     'positional': False,
+                     'required': False,
+                     'action': 'store',
+                     'nargs': 1,
+                     'help-text': arguments.FORMAT_HELP})
         return metadata
 
     def run_command(self, args, config):
-        print()
-        pull_all_manifest_repos(config['cfg_file'], config['user_cfg_file'])
-        print()
+        json_output = False
+        if args.format is not None:
+            if args.format[0] not in ['text', 'json']:

```

Comment:
This could be done using https://docs.python.org/3/library/argparse.html#choices it may be worth starting a discussion of the CLI functionality so that we can incorporate all of the features of arg parse. However this is beyond the scope of this change.